app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -986,7 +986,7 @@ $include-html-global-classes: $include-html-classes;
 // $include-html-panel-classes: $include-html-classes;
 
 // We use these to control the background and border styles
-// $panel-bg: scale-color($white, $lightness: -5%);
+$panel-bg: scale-color(#E6F5F4, $lightness: -5%);
```

Comment:
Color literals like `#E6F5F4` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E6F5F4` should be written as `#e6f5f4`